<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="number_of_scans_to_assemble_per_cloud" default="10" />
	<arg name="timeout_for_cloud_assembly" default="1.0" />


	<!-- LaserScan assembler -->
	<node pkg="laserscan_to_pointcloud" type="laserscan_to_pointcloud_assembler" name="laserscan_to_pointcloud_assembler" output="screen">
		<param name="laser_scan_topic" type="str" value="tilt_scan" />
		<param name="pointcloud_publish_topic" type="str" value="assembled_pointcloud" />
		<param name="number_of_scans_to_assemble_per_cloud" type="int" value="$(arg number_of_scans_to_assemble_per_cloud)" />
		<param name="timeout_for_cloud_assembly" type="double" value="$(arg timeout_for_cloud_assembly)" />
		<param name="target_frame" type="str" value="map" />
		<param name="min_range_cutoff_percentage_offset" type="double" value="5.00" />
		<param name="max_range_cutoff_percentage_offset" type="double" value="0.95" />
		<param name="include_laser_intensity" type="bool" value="false" />
		<param name="interpolate_scans" type="bool" value="false" />
	</node>
</launch>